src/interface.ts

```diff
@@ -33,6 +33,7 @@ export type RuleType =
   | 'boolean'
   | 'method'
   | 'regexp'
+  | 'array'
```

Comment:
looks like the array type is already defined here https://github.com/react-component/field-form/blob/master/src/interface.ts#L81